.github/workflows/capacitor-android.yml

```diff
@@ -51,21 +50,31 @@ jobs:
           base64 -d android/keystore.base64 > android/app.keystore
           rm android/keystore.base64
 
-      - name: Build and Sign Android App 🛠️
+      - name: Build and Sign Android App (APK) 🛠️
         run: |
           npx cap sync
           npx cap build android \
             --keystorepath $ANDROID_KEYSTORE_PATH \
             --keystorepass $ANDROID_KEYSTORE_PASSWORD \
             --keystorealias $ANDROID_KEYSTORE_ALIAS \
             --keystorealiaspass $ANDROID_KEYSTORE_ALIAS_PASSWORD \
-            --androidreleasetype $ANDROID_RELEASE_TYPE
+            --androidreleasetype APK
+
+      - name: Build and Sign Android App (AAB) 🛠️
+        run: |
+          npx cap build android \
+            --keystorepath $ANDROID_KEYSTORE_PATH \
+            --keystorepass $ANDROID_KEYSTORE_PASSWORD \
+            --keystorealias $ANDROID_KEYSTORE_ALIAS \
+            --keystorealiaspass $ANDROID_KEYSTORE_ALIAS_PASSWORD \
+            --androidreleasetype AAB
```

Comment:
Can you set the `ANDROID_RELEASE_TYPE` env for this specific task, so you don't need to modify the `scripts/capacitor/build-android.mjs`. Will it work? @dvelikiy